{u'description': [u"Donnie Berkholz's private developer overlay"],
 u'feeds': [u'https://cgit.gentoo.org/dev/dberkholz.git/atom/'],
 u'homepage': u'http://dev.gentoo.org/~dberkholz/overlay/',
 u'irc': None,
 u'name': u'dberkholz',
 u'official': False,
 u'owner_email': u'dberkholz@gentoo.org',
 u'owner_name': u'Donnie Berkholz',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'git://anongit.gentoo.org/dev/dberkholz', u'Git', u''),
              (u'https://anongit.gentoo.org/git/dev/dberkholz.git',
               u'Git',
               u''),
              (u'git+ssh://git@git.gentoo.org/dev/dberkholz.git',
               u'Git',
               u'')],
 u'status': u'unofficial',
 u'supported': True}
$ pmaint sync /home/mgorny/repos/dberkholz
WARNING:pkgcore:repository at location '/home/mgorny/repos/clickbeetle_tools' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/clickbeetle_tools', named '<unlabeled repository /home/mgorny/repos/clickbeetle_tools>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/armagetron' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/armagetron', named '<unlabeled repository /home/mgorny/repos/armagetron>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/arcon' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/arcon', named '<unlabeled repository /home/mgorny/repos/arcon>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/bicatali' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/bicatali', named '<unlabeled repository /home/mgorny/repos/bicatali>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/dauleet' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/dauleet', named '<unlabeled repository /home/mgorny/repos/dauleet>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/alice', named 'alice', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/Menelkir', named 'Menelkir', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/arcon' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/arcon', named '<unlabeled repository /home/mgorny/repos/arcon>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/dauleet' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/dauleet', named '<unlabeled repository /home/mgorny/repos/dauleet>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/armagetron', named 'armagetron', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/Menelkir', named 'Menelkir', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/clickbeetle_tools', named 'clickbeetle_tools', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/alice', named 'alice', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
Already up-to-date.
*** syncing '/home/mgorny/repos/dberkholz'...
*** synced '/home/mgorny/repos/dberkholz'
 * Sync succeeded
$ pmaint regen -t 2 dberkholz
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'

!!! ERROR: app-pda/ipheth-pair-9999 failed.

!!! ERROR: sci-biology/TMalign-9999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
 * 
 * "/home/mgorny/repos/dberkholz/sci-mathematics/pyquante/pyquante-1.6.3.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '190' 'depend'
 * 
 * "/home/mgorny/repos/dberkholz/sci-biology/biskit/biskit-2.3.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
 * 
 * "/home/mgorny/repos/dberkholz/sci-mathematics/elefant/elefant-0.4.0.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '176' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-biology/TMalign/TMalign-9999.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/app-pda/ipheth-pair/ipheth-pair-9999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-biology/TMalign/TMalign-9999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/app-pda/ipheth-pair/ipheth-pair-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-biology/TMalign/TMalign-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/app-pda/ipheth-pair/ipheth-pair-9999.ebuild'
!!!        ipheth-pair-9999.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!            TMalign-9999.ebuild, line   7:  Called inherit 'fortran'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'fortran'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for fortran failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: sci-biology/cytoscape-2.7.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sci-biology/ce-20040716 failed.

!!! ERROR: x11-drivers/nvidia-drivers-280.11 failed.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '195' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sci-biology/mmult-1.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-biology/cytoscape/cytoscape-2.7.0.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-biology/cytoscape/cytoscape-2.7.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-biology/cytoscape/cytoscape-2.7.0.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '227' 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         cytoscape-2.7.0.ebuild, line  11:  Called inherit 'versionator' 'java-pkg-2' 'java-ant-2' 'eutils'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/x11-drivers/nvidia-drivers/nvidia-drivers-280.11.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '171' 'depend'

!!! ERROR: app-emulation/vmware-modules-1.0.0.15-r3 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'versionator'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '165' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/x11-drivers/nvidia-drivers/nvidia-drivers-280.11.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-biology/ce/ce-20040716.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 161' for eclass versionator

received unknown com: export CATEGORY='media-gfx' PR=r0 PVR='2.0.0' PV='2.0.0' EBUILD='/home/mgorny/repos/dberkholz/media-gfx/fly/fly-2.0.0.ebuild' P='fly-2.0.0' PF='fly-2.0.0' PN=flypath
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/x11-drivers/nvidia-drivers/nvidia-drivers-280.11.ebuild'
!!!                  ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-biology/mmult/mmult-1.0.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-biology/ce/ce-20040716.ebuild'

!!! ERROR: dev-util/oprofile-0.9.3-r1 failed.
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   nvidia-drivers-280.11.ebuild, line   7:  Called inherit 'eutils' 'multilib' 'versionator' 'linux-mod' 'flag-o-matic' 'nvidia-driver'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-biology/ce/ce-20040716.ebuild'

!!! ERROR: net-wireless/iw-9999 failed.
!!!                  ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '250' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-biology/mmult/mmult-1.0.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'nvidia-driver'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-biology/mmult/mmult-1.0.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/nvidia-driver.eclass'
!!!                  ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ce-20040716.ebuild, line   7:  Called inherit 'toolchain-funcs'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               mmult-1.0.ebuild, line   5:  Called inherit 'autotools' 'eutils' 'fortran'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '153' 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '193' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'fortran'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/dev-util/oprofile/oprofile-0.9.3-r1.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/nvidia-driver.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain-funcs'
!!!                          ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/app-emulation/vmware-modules/vmware-modules-1.0.0.15-r3.ebuild'
!!!           nvidia-driver.eclass, line  16:  Called inherit 'readme.gentoo' 'versionator'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/net-wireless/iw/iw-9999.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for fortran failed


!!! ERROR: dev-lang/python-2.5.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                          ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/app-emulation/vmware-modules/vmware-modules-1.0.0.15-r3.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 211' for eclass toolchain-funcs

received unknown com: export CATEGORY='net-libs' PR=r0 PVR='0_p35005' PV='0_p35005' EBUILD='/home/mgorny/repos/dberkholz/net-libs/webkit-gtk/webkit-gtk-0_p35005.ebuild' P='webkit-gtk-0_p35005' PF='webkit-gtk-0_p35005' PN='webkit-gtk'path
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/dev-util/oprofile/oprofile-0.9.3-r1.ebuild'
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/app-emulation/vmware-modules/vmware-modules-1.0.0.15-r3.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'readme.gentoo'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/net-wireless/iw/iw-9999.ebuild'
!!!   vmware-modules-1.0.0.15-r3.ebuild, line   9:  Called inherit 'vmware-mod'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/net-wireless/iw/iw-9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '174' 'depend'

!!! ERROR: sci-libs/libxc-0.9 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/dev-util/oprofile/oprofile-0.9.3-r1.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/readme.gentoo.eclass'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vmware-mod'
!!!                 iw-9999.ebuild, line   7:  Called inherit 'git' 'toolchain-funcs' 'linux-info'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/readme.gentoo.eclass'
!!!           readme.gentoo.eclass, line  25:  Called die
!!! The die message:
!!!  Unsupported EAPI=2 (too old) for readme.gentoo

!!!                   ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vmware-mod failed


!!! ERROR: sci-libs/bigdft-1.2.0 failed.

!!! ERROR: app-misc/jq-1.3 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! die invoked from directory /usr/lib64/pkgcore
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/dev-lang/python/python-2.5.2.ebuild'
!!! Call stack:

!!! ERROR: sci-libs/etsf_io-1.0.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
 * 
 * "/home/mgorny/repos/dberkholz/sci-libs/pymc/pymc-2.0.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!       oprofile-0.9.3-r1.ebuild, line   5:  Called inherit 'eutils' 'qt3' 'linux-info'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'qt3'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for qt3 failed

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: app-laptop/tp_smapi-0.40-r3 failed.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/dev-lang/python/python-2.5.2.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '159' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: net-wireless/kismet-2007.10.1 failed.
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/dev-lang/python/python-2.5.2.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '144' 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '174' 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-libs/libxc/libxc-0.9.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!            python-2.5.2.ebuild, line  10:  Called inherit 'eutils' 'autotools' 'flag-o-matic' 'python' 'multilib' 'versionator' 'toolchain-funcs' 'alternatives' 'libtool'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'eutils'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-libs/libxc/libxc-0.9.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-libs/bigdft/bigdft-1.2.0.ebuild'
 * 
 * "/home/mgorny/repos/dberkholz/sci-chemistry/ccp1gui/ccp1gui-0.7.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '194' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-libs/bigdft/bigdft-1.2.0.ebuild'
 * 
 * "/home/mgorny/repos/dberkholz/sci-libs/mmtk/mmtk-2.5.23.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-libs/libxc/libxc-0.9.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 159' for eclass eutils

received unknown com: export CATEGORY='sci-libs' PR=r0 PVR='4.0.3' PV='4.0.3' EBUILD='/home/mgorny/repos/dberkholz/sci-libs/fox/fox-4.0.3.ebuild' P='fox-4.0.3' PF='fox-4.0.3' PN=foxpath
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/app-misc/jq/jq-1.3.ebuild'
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-libs/bigdft/bigdft-1.2.0.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '202' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/app-misc/jq/jq-1.3.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!            bigdft-1.2.0.ebuild, line   6:  Called inherit 'eutils' 'fortran' 'multilib'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '181' 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/app-misc/jq/jq-1.3.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/app-laptop/tp_smapi/tp_smapi-0.40-r3.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/net-wireless/kismet/kismet-2007.10.1.ebuild'
!!!                  jq-1.3.ebuild, line   7:  Called inherit 'autotools' 'eutils'

!!! ERROR: app-misc/labyrinth-9999 failed.
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!               libxc-0.9.ebuild, line   6:  Called inherit 'fortran' 'multilib'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'autotools'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/net-wireless/kismet/kismet-2007.10.1.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/app-laptop/tp_smapi/tp_smapi-0.40-r3.ebuild'

!!! ERROR: sci-chemistry/espresso-4.0_pre2 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'fortran'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 180' for eclass autotools

received unknown com: export CATEGORY='sci-libs' PR=r0 PVR=9999 PV=9999 EBUILD='/home/mgorny/repos/dberkholz/sci-libs/starparse/starparse-9999.ebuild' P='starparse-9999' PF='starparse-9999' PN=starparsepath
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
 * 
 * "/home/mgorny/repos/dberkholz/sci-chemistry/gridzilla/gridzilla-0.5_beta.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * 
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * "/home/mgorny/repos/dberkholz/sci-chemistry/cclib/cclib-0.91.ebuild":
 * 
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/app-laptop/tp_smapi/tp_smapi-0.40-r3.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-libs/etsf_io/etsf_io-1.0.2.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'fortran'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/net-wireless/kismet/kismet-2007.10.1.ebuild'
!!!        tp_smapi-0.40-r3.ebuild, line   7:  Called inherit 'eutils' 'linux-mod'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for fortran failed

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sci-libs/pgd-utils-0.4.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!! Call stack:
!!!  inherit for fortran failed

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-libs/etsf_io/etsf_io-1.0.2.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '180' 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-libs/etsf_io/etsf_io-1.0.2.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '191' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'linux-mod'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/app-misc/labyrinth/labyrinth-9999.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!        kismet-2007.10.1.ebuild, line   5:  Called inherit 'toolchain-funcs' 'linux-info' 'eutils'

!!! ERROR: sci-libs/starparse-9999_pre09 failed.
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '209' 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass linux-mod

received unknown com: export CATEGORY='sci-libs' PR=r0 PVR='1.1' PV='1.1' EBUILD='/home/mgorny/repos/dberkholz/sci-libs/wannier90/wannier90-1.1.ebuild' P='wannier90-1.1' PF='wannier90-1.1' PN=wannier90path
!!!           etsf_io-1.0.2.ebuild, line   6:  Called inherit 'autotools' 'eutils' 'fortran' 'flag-o-matic' 'multilib'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'linux-info'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/app-misc/labyrinth/labyrinth-9999.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass linux-info

received unknown com: export CATEGORY='dev-java' PR=r0 PVR='0.92_beta' PV='0.92_beta' EBUILD='/home/mgorny/repos/dberkholz/dev-java/fop/fop-0.92_beta.ebuild' P='fop-0.92_beta' PF='fop-0.92_beta' PN=foppath
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-libs/pgd-utils/pgd-utils-0.4.2.ebuild'
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-libs/pgd-utils/pgd-utils-0.4.2.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-libs/pgd-utils/pgd-utils-0.4.2.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-chemistry/espresso/espresso-4.0_pre2.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-chemistry/espresso/espresso-4.0_pre2.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'fortran'
!!!         pgd-utils-0.4.2.ebuild, line   5:  Called inherit 'distutils'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/app-misc/labyrinth/labyrinth-9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '214' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!          labyrinth-9999.ebuild, line   8:  Called inherit 'gnome2' 'autotools' 'subversion'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for fortran failed


!!! ERROR: sci-libs/vcg-9999 failed.
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-chemistry/espresso/espresso-4.0_pre2.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 164' for eclass distutils

received unknown com: export CATEGORY='sci-chemistry' PR=r0 PVR='2.8' PV='2.8' EBUILD='/home/mgorny/repos/dberkholz/sci-chemistry/namd/namd-2.8.ebuild' P='namd-2.8' PF='namd-2.8' PN=namdpath
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-pda/ipheth-pair-9999
!!! caught exception inherit requires an unknown eclass, fortran cannot be found while processing ebuild src: sci-biology/TMalign-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-biology/cytoscape-2.7.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-biology/ce-20040716
!!! caught exception inherit requires an unknown eclass, fortran cannot be found while processing ebuild src: sci-biology/mmult-1.0
!!! caught exception inherit requires an unknown eclass, vmware-mod cannot be found while processing ebuild src: app-emulation/vmware-modules-1.0.0.15-r3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: media-gfx/fly-2.0.0
!!! caught exception inherit requires an unknown eclass, qt3 cannot be found while processing ebuild src: dev-util/oprofile-0.9.3-r1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: net-wireless/iw-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/python-2.5.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/webkit-gtk-0_p35005
!!! caught exception inherit requires an unknown eclass, fortran cannot be found while processing ebuild src: sci-libs/libxc-0.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-drivers/nvidia-drivers-280.11
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-misc/jq-1.3
!!! caught exception inherit requires an unknown eclass, fortran cannot be found while processing ebuild src: sci-libs/bigdft-1.2.0
!!! caught exception inherit requires an unknown eclass, fortran cannot be found while processing ebuild src: sci-libs/etsf_io-1.0.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-laptop/tp_smapi-0.40-r3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-wireless/kismet-2007.10.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-libs/fox-4.0.3
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sci-chemistry/imosflm-1.0.5
!!! caught exception inherit requires an unknown eclass, fortran cannot be found while processing ebuild src: sci-chemistry/espresso-4.0_pre2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-libs/starparse-9999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sci-chemistry/mole-bin-1.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-libs/pgd-utils-0.4.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-libs/starparse-9999_pre09
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-libs/wannier90-1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-java/fop-0.92_beta
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sci-chemistry/pymss-0.8.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-libs/vcg-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-chemistry/namd!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-libs/starparse/starparse-9999_pre09.ebuild'
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!       espresso-4.0_pre2.ebuild, line   7:  Called inherit 'autotools' 'eutils' 'fortran' 'toolchain-funcs'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'fortran'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for fortran failed

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-libs/starparse/starparse-9999_pre09.ebuild'

!!! ERROR: sci-chemistry/zodiac-0.5 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: www-apps/trac-git-9999 failed.
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-libs/starparse/starparse-9999_pre09.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI= is not supported

!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '174' 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '150' 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!    starparse-9999_pre09.ebuild, line  12:  Called inherit 'autotools' 'bzr'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'bzr'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 180' for eclass bzr

received unknown com: export CATEGORY='sci-chemistry' PR=r0 PVR=20060606 PV=20060606 EBUILD='/home/mgorny/repos/dberkholz/sci-chemistry/pmb/pmb-20060606.ebuild' P='pmb-20060606' PF='pmb-20060606' PN=pmbpath
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-chemistry/zodiac/zodiac-0.5.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/sci-libs/vcg/vcg-9999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-libs/vcg/vcg-9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '177' 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-libs/vcg/vcg-9999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/sci-chemistry/zodiac/zodiac-0.5.ebuild'
 * Use PYTHON_DEPEND variable instead of NEED_PYTHON variable.

!!! ERROR: dev-python/multitask-0.2.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                vcg-9999.ebuild, line   8:  Called inherit 'cvs'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/sci-chemistry/zodiac/zodiac-0.5.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/aggdraw/aggdraw-1.2_alpha3.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/www-apps/trac-git/trac-git-9999.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/jpype/jpype-0.5.4.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
!!!              zodiac-0.5.ebuild, line   5:  Called inherit 'versionator' 'qt4'
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cvs'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/www-apps/trac-git/trac-git-9999.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 194' for eclass cvs

received unknown com: export CATEGORY='sci-chemistry' PR=r0 PVR='0.2.17' PV='0.2.17' EBUILD='/home/mgorny/repos/dberkholz/sci-chemistry/shakerr/shakerr-0.2.17.ebuild' P='shakerr-0.2.17' PF='shakerr-0.2.17' PN=shakerrpath
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/www-apps/trac-git/trac-git-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'qt4'
!!!           trac-git-9999.ebuild, line   5:  Called inherit 'git' 'distutils'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for qt4 failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '193' 'depend'
 * Use PYTHON_DEPEND variable instead of NEED_PYTHON variable.

!!! ERROR: dev-python/pycallgraph-0.4.1 failed.
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 206' for eclass git

received unknown com: export CATEGORY='app-shells' PR=r0 PVR=050706 PV=050706 EBUILD='/home/mgorny/repos/dberkholz/app-shells/heirloom-sh/heirloom-sh-050706.ebuild' P='heirloom-sh-050706' PF='heirloom-sh-050706' PN='heirloom-sh'failed
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/python-biggles/python-biggles-1.6.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/rdfalchemy/rdfalchemy-0.2_beta2.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/dev-python/multitask/multitask-0.2.0.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/dev-python/pycallgraph/pycallgraph-0.4.1.ebuild'
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/spasmoidal/spasmoidal-0.1.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/dev-python/multitask/multitask-0.2.0.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/dev-python/pycallgraph/pycallgraph-0.4.1.ebuild'
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/statlib/statlib-1.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/dev-python/multitask/multitask-0.2.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/dev-python/pycallgraph/pycallgraph-0.4.1.ebuild'

!!! ERROR: x11-libs/agg-2.5-r3 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         multitask-0.2.0.ebuild, line   8:  Called inherit 'distutils'
!!!       pycallgraph-0.4.1.ebuild, line   7:  Called inherit 'distutils' 'eutils'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!              ebuild-daemon.lib, line  23:  Called __qa_invoke '__preloaded_eclass_distutils'
!!!              ebuild-daemon.lib, line  23:  Called __qa_invoke '__preloaded_eclass_distutils'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called __preloaded_eclass_distutils
!!!             ebuild-daemon.bash, line 394:  Called inherit 'python'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '158' 'depend'
!!!         isolated-functions.lib, line 221:  Called __preloaded_eclass_distutils
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'python'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
 * 
 * "/home/mgorny/repos/dberkholz/dev-python/suds/suds-0.4.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!              ebuild-daemon.lib, line  23:  Called __qa_invoke '__preloaded_eclass_python'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/dberkholz/x11-libs/agg/agg-2.5-r3.ebuild'
!!!             ebuild-daemon.bash, line 394:  Called inherit 'python'
!!!         isolated-functions.lib, line 221:  Called __preloaded_eclass_python
!!!             ebuild-daemon.bash, line 656:  Called die
!!! The die message:
!!!  NEED_PYTHON variable is banned

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/dberkholz/x11-libs/agg/agg-2.5-r3.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/dberkholz/x11-libs/agg/agg-2.5-r3.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'python'
!!!              agg-2.5-r3.ebuild, line   7:  Called inherit 'eutils' 'autotools' 'base'
!!!              ebuild-daemon.lib, line  23:  Called __qa_invoke '__preloaded_eclass_python'
!!!         isolated-functions.lib, line 221:  Called __preloaded_eclass_python
!!!             ebuild-daemon.bash, line 656:  Called die
!!! The die message:
!!!  NEED_PYTHON variable is banned

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'autotools'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 177' for eclass autotools

received unknown com: export CATEGORY='sys-power' PR=r0 PVR=9999 PV=9999 EBUILD='/home/mgorny/repos/dberkholz/sys-power/powertop/powertop-9999.ebuild' P='powertop-9999' PF='powertop-9999' PN=powertoppath
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

 * 
 * "/home/mgorny/repos/dberkholz/app-office/rescuetime-linux-uploader/rescuetime-linux-uploader-99.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
-2.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-chemistry/mage-6.44.060606
!!! caught exception inherit requires an unknown eclass, qt4 cannot be found while processing ebuild src: sci-chemistry/zodiac-0.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-shells/heirloom-sh-050706
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-misc/labyrinth-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-chemistry/pmb-20060606
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: dev-python/cocos2d-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-chemistry/shakerr-0.2.17
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: dev-python/pymedia-1.3.7.3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-doc/bashdoc-1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: media-libs/libv4l-0.5.8
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: dev-python/requests-0.8.3
!!! caught exception Metadata Exception: pkg ebuild src: x11-libs/agg-2.5-r3, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: x11-libs/agg-2.5-r3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/pycallgraph-0.4.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/multitask-0.2.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-power/powertop-9999
Error running an operation:
 Unhandled exception in regen_cache
Sandboxed process killed by signal: Broken pipe
 * Cache regen failed with 246
